dap_aria_mapping/pipeline/label_docs_with_taxonomy.py

```diff
@@ -0,0 +1,103 @@
+from dap_aria_mapping.getters.taxonomies import get_cooccurrence_taxonomy, get_semantic_taxonomy
+from dap_aria_mapping.getters.openalex import get_openalex_entities
+from dap_aria_mapping.getters.patents import get_patent_entities
+from collections import defaultdict
+from dap_aria_mapping import BUCKET_NAME
+from nesta_ds_utils.loading_saving.S3 import upload_obj
+import pandas as pd
+import json
+import argparse
+
+def tag_docs_at_level(docs: dict, tax_at_level: pd.Series) -> dict:
+    """tag documents with topic labels based on entities present in document
+
+    Args:
+        docs (dict): dictionary where key: document id, value: list of entity information about each document
```

Comment:
I'd say more PEP-compliant hints would be `Dict[str, Sequence[str]]`. In addition, I agree sometimes it's better to be more verbose, but I'd recommend writing instead:
        `{
            article_id: [
                topic_id,
                topic_id,
                ...
            ]
            ...
        }`

Reply:
done